### PR TITLE
Add examples with `type: file` for request and responses.

### PR DESCRIPTION
**Pull request by kylef**

### CODE REVIEW

File: fixtures/features/api-elements/form-data-file.json [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+                      "content": 224
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      },
+      "content": "Unable to generate multipart/form-data; boundary=BOUNDARY example message body out of JSON Schema"
```
Comment (kylef) [inline fixtures/features/api-elements/form-data-file.json:344]: This was existing behaviour in this case, there is no existing test. Unfortunately we will likely need to handle remote referencing and embedding binary examples to solve this. I think a larger task to be tackled separately.

Comment (kylef) [reply fixtures/features/api-elements/form-data-file.json:344]: Note, if the user has non-referenced string example it would already work.

**Comment by pksunkara** [comment]
Where is the produces-file.yaml?

**Comment by kylef** [comment]
Whoops, added missing `produces-file.yaml` @pksunkara

**Review by pksunkara — APPROVED** [review]

**Pull request merged**